homeassistant/components/ambient_network/manifest.json

```
@@ -0,0 +1,11 @@
+{
+  "domain": "ambient_network",
+  "name": "Ambient Weather Network",
+  "codeowners": ["@thomaskistler"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/ambient_network",
+  "integration_type": "hub",
```

I think this is considered a [service integration](https://developers.home-assistant.io/blog/2022/10/24/hubs-devices-services-virtual-integrations?_highlight=integration_type#differentiating-hubs-devices-and-services)
```suggestion
  "integration_type": "service",
```

---

Fixed.